src/components v2/Forms/Atoms/Label/Label.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="photon-label">
+    <span class="label">{{ label }}</span>
+    <span v-if="isRequired" class="required-astrix">*</span>
+    <span class="inline-text">{{ inlineText }}</span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'P2Label',
+  props: {
+    label: {
+      type: String,
+      default: '',
+    },
+    isRequired: {
+      type: Boolean,
+      default: false,
+    },
+    inlineText: {
+      type: String,
+      default: '',
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.photon-label {
+  display: flex;
+  align-items: baseline;
+  margin: var(--label-base-margin, 0 0 0.5rem 0);
+
+  .label {
+    font-family: var(--label-base-font-family, "'roboto' arial sans-serif");
+    font-weight: var(--label-base-font-weight, bold);
+    font-size: var(--label-base-font-size, 1rem);
```

Comment:
Are switching to `px` instead of `rem`

Reply:
for fonts? I thought we switched to PX for spacing only.